docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+services:
+  frozen_flame:
+    container_name: frozenFlame
```

Comment:
typical naming convention here is usually hyphenated, so maybe this could be `frozen-flame-dedicated-server` or even just `frozen-flame-server`. Similar change to the service name here on L3

Reply:
I let you for the naming convention. I am not the best for that. But I prefer `frozen-flame-server`